data/json/mapgen/parks_recreation/fishing_pond0_0.json

```diff
@@ -0,0 +1,87 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "fishing_pond_0_0" ],
+    "weight": 100,
+    "object": {
+      "rows": [
+        "||||||||||||||||||||||||",
+        "|.......................",
+        "|..1...f...#.......4....",
+        "|t............1....#....",
+        "|....4....#.......#d#s..",
+        "|..._bb_________________",
+        "|#.&____________________",
+        "|..________#.........WWW",
+        "|.._______#..4.#..iWWWWW",
+        "|.________..f...WWWWWWww",
+        "|.________4....iiWWWwwww",
+        "|.___#____.#..iWWWwcwwww",
+        "|.__4..#___..WWWWwwwwwww",
+        "|.__.#..____pppppppppp-w",
+        "|d__....4__ppppppppppp-w",
+        "|.___u.....ppppppppppp-w",
+        "|.___.....iWWwwwwwwwpp-w",
+        "|.___.1..WiWwwwwwwwwpp-w",
+        "|.___#..iiWWwwwwwwwwpp-w",
+        "|#.___#.WWWwwwwwwwwwpp-w",
+        "|..___#iWWwcwwwwwwww---w",
+        "|..___.WiWwwwwwwwwwwwwww",
+        "|.4___.WWwwwwwwwwwwwwwww",
+        "|...__#WWwwwwwwwwwwwwwww"
+      ],
+      "terrain": {
+        "#": "t_underbrush_harvested_autumn",
+        "&": "t_dirt",
+        "-": "t_railing_v",
+        ".": "t_grass",
+        "1": "t_tree_young",
+        "4": "t_tree_pine",
+        "W": "t_water_sh",
+        "_": "t_dirt",
+        "b": "t_dirt",
+        "c": "t_water_dp",
+        "d": "t_grass",
+        "f": "t_grass",
+        "i": "t_water_sh",
+        "p": "t_pavement",
+        "s": "t_shrub_strawberry",
+        "t": "t_tree_deadpine",
+        "u": "t_underbrush",
+        "w": "t_water_dp",
+        "|": "t_chainfence_h"
+      },
+      "furniture": {
+        "#": "f_null",
+        "&": "f_trashcan",
+        "-": "f_null",
+        ".": "f_null",
+        "1": "f_null",
+        "4": "f_null",
+        "W": "f_null",
+        "_": "f_null",
+        "b": "f_bench",
+        "c": "f_cattails",
+        "d": "f_dandelion",
+        "f": "f_bluebell",
+        "i": "f_cattails",
+        "p": "f_null",
+        "s": "f_null",
+        "t": "f_null",
+        "u": "f_null",
+        "w": "f_null",
+        "|": "f_null"
+      },
+      "place_monster": [
+        { "monster": "mon_rabbit", "x": 8, "y": 4 },
+        { "monster": "mon_squirrel_red", "x": 2, "y": 9 },
+        { "monster": "mon_fish_lbass", "x": 22, "y": 11 },
+        { "monster": "mon_duck", "x": 9, "y": 16 },
+        { "monster": "mon_duck", "x": 8, "y": 17 },
+        { "monster": "mon_fish_sbass", "x": 15, "y": 21 },
+        { "monster": "mon_fish_bluegill", "x": 21, "y": 23 }
+      ]
```

Comment:
Placing monsters by hand like that is pretty ugly. It's not necessary to rework this here, but they should be wrapped in a group and spawned as entries from the group rather than being hand-picked and completely identical every time.

Reply:
I fully agree.